File: solutions/217-contains-duplicate/217 Contains Duplicate.swift
class Solution {
    func containsDuplicate(_ nums: [Int]) -> Bool {
        var nums_set = Set<Int>()
        for i in nums {
            if nums_set.contains(i) {
                return true
            } else {
                nums_set.insert(i)
            }
        }
        return false
    }
}

let bruh = Solution()
let list = [1,2,3,4,5,6,7,8,9,10]
print(bruh.containsDuplicate(list))

File: solutions/217-contains-duplicate/217 Contains Duplicate.py
class Solution(object):
    def __self__(self):
        pass

    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        nums_set = set()
        for i in nums:
            if i in nums_set:
                return True
            else:
                nums_set.add(i)
        return False


nums2 = [1, 2, 3, 4]
bruh = Solution()
print(bruh.containsDuplicate(nums2))
